app/stores/ada/DaedalusTransferStore.js

```diff
@@ -119,14 +121,20 @@ export default class DaedalusTransferStore extends Store {
     });
 
     this.ws.addEventListener('close', (event: any) => {
-      Logger.info(
-        `[ws::close] CODE: ${event.code} - REASON: ${event.reason} - was clean? ${event.wasClean}`
-      );
       if (event.code !== WS_CODE_NORMAL_CLOSURE) {
+        // if connection was not closed normally, we log this as an error. Otherwise it's an info
+        Logger.error(
+          `[ws::close] CODE: ${event.code} - REASON: ${event.reason} - was clean? ${event.wasClean}`
```

Comment:
the two logs are equal..I guess the idea was to differentiate the logs right?

Reply:
yeah the idea is just that the log level changes. You could avoid duplication a bit by having a variable for the log level and then one log statement that just changes the level

Reply:
oh I never saw that one was `.error` and the other `.info`, my bad,  :+1: